dockerfiles/web/Dockerfile

```diff
@@ -7,8 +8,9 @@ ENV DEBIAN-FRONTEND noninteractive
 ENV DISPLAY=:1
 ENV GECKODRIVER_VERSION="v0.26.0"
 RUN echo "deb http://deb.debian.org/debian stretch-backports main" > /etc/apt/sources.list.d/backports.list
-RUN wget -O - https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get update && apt-get install -y xvfb firefox-esr libpq-dev python3-dev nodejs npm && apt-get install -y -t stretch-backports libsqlite3-0 && apt-get clean
+RUN wget -O - https://deb.nodesource.com/setup_12.x | bash -
+RUN apt-get update && apt-get install -y xvfb firefox-esr libpq-dev python3-dev nodejs && \
```

Comment:
`npm` was removed intentionally?

Reply:
Correct: `npm` is now along for the ride when you install `nodejs` from nodesource, and if you try to install it it `CONFLICTS nodejs` and apt yells at you for holding broken packages